recipes/fmtlog/all/patches/2.2.1-0001-fix-cmake.patch

```diff
@@ -0,0 +1,39 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
Please submit this upstream!

Reply:
It seems not needed for upstream because upstream uses internal fmtlib.